settings.gradle.kts

```diff
@@ -37,7 +37,15 @@ dependencyResolutionManagement {
             version("com.google.protobuf", "3.24.0")
             version("io.helidon.webserver.http2", "4.0.10")
             version("io.helidon.webserver.grpc", "4.0.10")
-
+            version("com.swirlds.metrics.api","0.51.5")
+            version("com.swirlds.metrics.impl","0.51.5")
+            version("com.swirlds.common", "0.51.5")
+            version("com.swirlds.config.impl", "0.51.5")
+            version("com.swirlds.config.processor", "0.51.5")
+            version("com.swirlds.config.extensions", "0.51.5")
```

Comment:
Are these versions ^^ always in lock-step?  Maybe we can set a `var metricsVersion = "0.51.5"` above the `dependencyResolutionManagement` block?

Reply:
Good Idea! 👍 
I think they are in lockstep indeed!

Reply:
✅ Done!